sherpa/astro/ui/utils.py

```diff
@@ -12105,6 +12167,78 @@ def plot_arf(self, id=None, resp_id=None, replot=False, overplot=False,
         self._plot(plotobj, overplot=overplot, clearwindow=clearwindow,
                    **kwargs)
 
+
+    def plot_rmf(self, id=None, resp_id=None, replot=False, overplot=False,
+                 clearwindow=True, **kwargs):
+        """Plot the RMF associated with a data set.
+
+        Display the energy redistribution from the RMF
+        component of a PHA data set.
+
```

Comment:
```suggestion

        .. vresionadded:: 4.16.0
        
```

Again, is indentation correct?

Reply:
`versionadded` also needs to be spelled correctly.

Reply:
Added your comment, so that I could add it to batch directly...